[P0] Get-IPv4ViaKVP : 8 hours delay in aborting single test

```

05/05/2019 11:51:45 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 11:56:57 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 12:02:08 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 12:07:19 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 12:12:30 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 12:17:41 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 12:22:52 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 12:28:03 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 12:33:13 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 12:38:25 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 12:43:35 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 12:48:46 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 12:53:57 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 12:59:08 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 13:04:19 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 13:09:30 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 13:14:41 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 13:19:52 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 13:25:03 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 13:30:14 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 13:35:25 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 13:40:36 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 13:45:47 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 13:50:58 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 13:56:09 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 14:01:20 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 14:06:31 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 14:11:42 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 14:16:53 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 14:22:04 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 14:27:15 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 14:32:26 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 14:37:37 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 14:42:48 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 14:47:59 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 14:53:10 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 14:58:21 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 15:03:32 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 15:08:43 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 15:13:54 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 15:19:05 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 15:24:16 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 15:29:27 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 15:34:38 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 15:39:49 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 15:45:00 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 15:50:11 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 15:55:22 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 16:00:33 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 16:05:44 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 16:10:55 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 16:16:06 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 16:21:17 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 16:26:28 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 16:31:39 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 16:36:50 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 16:42:01 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 16:47:12 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 16:52:23 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 16:57:34 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 17:02:45 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 17:07:56 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 17:13:07 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 17:18:18 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 17:23:29 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 17:28:40 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 17:33:51 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 17:39:02 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 17:44:13 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 17:49:24 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 17:54:35 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 17:59:46 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 18:04:57 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 18:10:08 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 18:15:19 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 18:20:30 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 18:25:41 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 18:30:52 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 18:36:03 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 18:41:14 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 18:46:25 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 18:51:36 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 18:56:47 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 19:01:58 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 19:07:09 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 19:12:20 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 19:17:31 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 19:22:42 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 19:27:53 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 19:33:04 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 19:38:15 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 19:43:26 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 19:48:37 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 19:53:48 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 19:58:59 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 20:04:10 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 20:09:21 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 20:14:32 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 20:19:43 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 20:24:54 : [WARN ] Get-IPv4ViaKVP: No IPv4 address found for VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0

05/05/2019 20:25:04 : [ERROR] Wait-ForVMToStartKVP: VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0 did not start KVP within timeout period (1000)

05/05/2019 20:25:04 : [ERROR] Get-IPv4AndWaitForSSHStart: Unable to get ipv4 from VM LISAv2-OneVM-centos_7.2_x64-d-284-EW29-636926271935-role-0 via KVP within timeout period (1000)

```

--- a/Testscripts/Windows/KDUMP-TestScript.ps1
+++ b/Testscripts/Windows/KDUMP-TestScript.ps1
@@ -74,7 +74,7 @@ function Main {
             }
 
             $newIP = Get-IPv4AndWaitForSSHStart -VMName $VMName -HvServer $HvServer `
-                -VmPort $VmPort -User $VMUserName -Password $VMPassword -StepTimeout 360
+                -VmPort $VmPort -User $VMUserName -Password $VMPassword -StepTimeout 30
             if ($newIP) {
                 $vm2ipv4 = $newIP
             } else {

--- a/Testscripts/Windows/Max-vCPU.ps1
+++ b/Testscripts/Windows/Max-vCPU.ps1
@@ -86,7 +86,7 @@ function Main {
     # Start VM and wait for SSH access
     Start-VM -Name $VMName -ComputerName $HvServer
     $newIpv4 = Get-Ipv4AndWaitForSSHStart $VMName $HvServer $VMPort $VMUserName `
-                $VMPassword 300
+                $VMPassword 30
     if ($newIpv4) {
         # In some cases the IP changes after a reboot
         Set-Variable -Name "Ipv4" -Value $newIpv4

--- a/Testscripts/Windows/PRODUCTION-CHECKPOINT-3CHAIN-VHD.ps1
+++ b/Testscripts/Windows/PRODUCTION-CHECKPOINT-3CHAIN-VHD.ps1
@@ -207,7 +207,7 @@ function Main {
         Write-LogInfo "Get the new IPv4 for $vmNameChild"
         # Get new IPV4
         $newIP = Get-IPv4AndWaitForSSHStart -VMName $vmNameChild -HvServer $hvServer `
-                -VmPort $port -User $user -Password $password -StepTimeout 360
+                -VmPort $port -User $user -Password $password -StepTimeout 30
         if ($newIP) {
             $vm2ipv4 = $newIP
         } else {

--- a/Testscripts/Windows/Reboot-Diff-Memory-Settings.ps1
+++ b/Testscripts/Windows/Reboot-Diff-Memory-Settings.ps1
@@ -79,7 +79,7 @@ function Main {
 
         $Error.Clear()
         # If the VM has no IP it means it rebooted
-        $new_ipv4 = Get-IPv4AndWaitForSSHStart $VMName $HvServer $VMPort $VMUserName $VMPassword 300
+        $new_ipv4 = Get-IPv4AndWaitForSSHStart $VMName $HvServer $VMPort $VMUserName $VMPassword 30
         if ($new_ipv4) {
             # In some cases the IP changes after a reboot
             Write-LogInfo "${VMName} IP Address after reboot: ${new_ipv4}"

--- a/Testscripts/Windows/SETUP-STOR-TakeRevert-Snapshot.ps1
+++ b/Testscripts/Windows/SETUP-STOR-TakeRevert-Snapshot.ps1
@@ -114,7 +114,7 @@ function Main {
     Start-VM $VMName -ComputerName $HvServer
 
     $newIpv4 = Get-Ipv4AndWaitForSSHStart -VmName $VMName -HvServer $HvServer -Vmport $VMPort -User $VMUserName `
-              -Password $VMPassword -StepTimeout 300
+              -Password $VMPassword -StepTimeout 30
     if (-not $newIpv4) {
         Write-LogErr "Failed to retrive $VMName IP after snasphot restore"
         return "FAIL"

--- a/Testscripts/Windows/SRIOV-REBOOT-VM.ps1
+++ b/Testscripts/Windows/SRIOV-REBOOT-VM.ps1
@@ -23,7 +23,7 @@ function Main {
         $VMPassword
     )
 
-    $timeout = 300
+    $timeout = 30
 
     # Get IP
     $ipv4 = Get-IPv4ViaKVP $VMName $HvServer

--- a/Testscripts/Windows/STRESS-MTU-Netvsc-Reload.ps1
+++ b/Testscripts/Windows/STRESS-MTU-Netvsc-Reload.ps1
@@ -67,7 +67,7 @@ function Main {
 
     Start-Sleep -s 600
     $NewIP = Get-IPv4AndWaitForSSHStart -VmName $VMName -HvServer $HvServer -Vmport $VMPort `
-        -Password $VMPassword -User $VMUserName -StepTimeout 1000
+        -Password $VMPassword -User $VMUserName -StepTimeout 30
         $allVmData.PublicIP = $NewIP
     if (-not $NewIP) {
         Write-LogErr "VM became unresponsive after reloading hv_netvsc"

